packages/bahmni-widgets/src/forms/__tests__/FormsTable.test.tsx

```
@@ -1241,15 +1246,11 @@ describe('FormsTable', () => {
-      await waitFor(
-        () => {
-          expect(mockGetPatientFormData.mock.calls).toHaveLength(
-            initialCallCount,
-          );
-        },
-        { timeout: 500 },
-      );
+      await waitFor(() => {
+        expect(
+          mockGetPatientFormData.mock.calls.length,
+        ).toBeGreaterThan(initialCallCount);
```

ESLint

[error] Replace `⏎··········mockGetPatientFormData.mock.calls.length,⏎········).toBeGreaterThan(initialCallCount` with `mockGetPatientFormData.mock.calls.length).toBeGreaterThan(⏎··········initialCallCount,⏎········`

(prettier/prettier)